static/js/composebox_typeahead.js

```diff
@@ -323,17 +328,22 @@ exports.tokenize_compose_str = function (s) {
         case '~':
             // Code block must start on a new line
             if (i === 2) {
-                return s.slice(0);
```

Comment:
Probably this should go in a separate commit that what's the context of changing this as it seems to be unrelated from the PR's goal?

Reply:
It's also worth mentioning that this change could have a functional effect, e.g. if part of the purpose here was the make a copy of the string, I think `s.slice(0)` would do that.

Reply:
https://stackoverflow.com/questions/5024085/whats-the-point-of-slice0-here